connectors/email/src/test/java/io/camunda/connector/email/client/jakarta/TestMessage.java

```diff
@@ -207,17 +204,28 @@ public void addHeader(String header_name, String header_value) throws MessagingE
   public void removeHeader(String header_name) throws MessagingException {}
 
   @Override
-  public Enumeration<Header> getAllHeaders() throws MessagingException {
-    return null;
+  public Enumeration<jakarta.mail.Header> getAllHeaders() throws MessagingException {
+    return new Enumeration<>() {
+      @Override
+      public boolean hasMoreElements() {
+        return false;
+      }
+
+      @Override
+      public jakarta.mail.Header nextElement() {
+        return null;
+      }
+    };
   }
 
   @Override
-  public Enumeration<Header> getMatchingHeaders(String[] header_names) throws MessagingException {
+  public Enumeration<jakarta.mail.Header> getMatchingHeaders(String[] header_names)
```

Comment:
Is using the fully qualified `jakarta.mail.Header` intentional here and elsewhere in this file?

Reply:
I removed the fully qualified class names